pkg/kapp/logger/ui.go

```diff
@@ -11,7 +11,9 @@ import (
 )
 
 const (
+	// nolint: deadcode,unused
```

Comment:
i think we should use these below in Info and Error methods.

Reply:
Using in associated methods now.